akumulid/tcp_server.cpp

```diff
@@ -2,6 +2,7 @@
 #include "utility.h"
 #include <thread>
 #include <atomic>
+#include <unistd.h>
```

Comment:
akumuli_def.h contains it's own set of integer types that is used across the project - u8, u16, u32, u64 and signed i8, i16, i32, i64. And in the C++ code it's better to use <cunistd> instead of <unistd.h>.